tests/lib/test_schema.py

```diff
@@ -0,0 +1,151 @@
+import yaml
+import sys
+import pprint
+
+def check_bool(value, expected):
+    if expected == 'false()' and value is False:
+        return 1
```

Comment:
Any reason to return 1/0 instead of true/false?

Reply:
I did as I was used from Perl. We don't have booleans there ;-)
I couldn't get it to work with True/False, though, maybe I had a typo